$NetBSD: patch-aa,v 1.7 2006/02/06 19:12:24 joerg Exp $

This patch is a context diff to hack around bugs in GNU patch.

*** sys/tryclockgettime.c.orig	Wed Feb 22 14:38:44 2006
--- sys/tryclockgettime.c	Wed Feb 22 14:40:04 2006
***************
*** 2,3 ****
--- 2,4 ----
+ #include <sys/time.h>
  #include <time.h>
  
  int main(void)
